openhands/server/session/session.py

```diff
@@ -253,7 +253,7 @@ async def dispatch(self, data: dict):
                     return
         self.agent_session.event_stream.add_event(event, EventSource.USER)
 
-    async def send(self, data: dict[str, object]):
+    async def send(self, data: dict[str, object]) -> None:
```

Comment:
```suggestion
    async def send(self, data: dict[str, object]) -> bool:
```

I think the method called at the end makes this `bool` return type